### DISCUSSION TITLE
SMTP smuggling

### DISCUSSION THREAD

**Discussion by colinleroy** [Mailu/Mailu#3112] [General]
Hello,
I suppose Mailu's postfix container is, by default, susceptible to SMTP smuggling as described here: https://www.postfix.org/smtp-smuggling.html
Is there a timeline on an update?
Thanks in advance!

**Comment by nextgens** [comment]
You are supposing wrong; [2.0.35](https://github.com/Mailu/Mailu/releases/tag/2.0.35) has a workaround and [2.0.36](https://github.com/Mailu/Mailu/releases/tag/2.0.36) the long term fix.

**Reply by colinleroy** [reply]
Oh, I'm sorry, I didn't think of looking in github's releases, only on mailu.io's release notes!
Thanks!
